pkg/sync/v6/data/ClusterRole.rbac.authorization.k8s.io/self-provisioner.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   annotations:
     authorization.openshift.io/system-only: "true"
     openshift.io/description: A user that can request projects.
+    openshift.io/reconcile-protect: "true"
```

Comment:
why this change, out of interest?

Reply:
Because internal OCP reconcile loop keeps defaulting them back :/ we need to remove it from the reconcide loop 
https://access.redhat.com/solutions/3304341

Reply:
still don't get it - is this happening on clusters today?  If not, what changes here to make it a problem?

Reply:
yes, it had to be on ClusterRoleBinding only. Not here. but it was 2 weeks when I was working with it. WIll retest and remove if this is a case.